hyperbook/book/allgemein/index.md

```diff
@@ -50,8 +50,8 @@ Das Anmeldeverfahren sieht etwa wie folgt aus. Genaue Termine des
 aktuellen Zeitraumes finden sich auf der [Website unter
 Ablauf und Termine](https://software-challenge.de/ablauf-und-termine).
 
-Ab April können interessierte SchulleiterInnen, Lehrkräfte und
-SchülerInnen ihre Schulen und Teams für die Teilnahme an der
+Ab April können interessierte Schulleiterinnen, Schulleiter, Lehrkräfte,
```

Comment:
Wouldn't it make sense to just name the *Lehrkräfte* and the *SuS*? After all, the principals are usually teachers, too.

Reply:
Fair, I just directly adapted the text here, but I might as well change that

Reply:
![sus](https://user-images.githubusercontent.com/19777592/187048194-495577a5-28ea-428f-ae90-6d7dc117c717.jpg)